UnitTests/CardPaymentsTests/MockCardVaultDelegate.swift

```diff
@@ -0,0 +1,25 @@
+@testable import CorePayments
+@testable import CardPayments
+
+class MockCardVaultDelegate: CardVaultDelegate {
```

Comment:
This is awesome, nice job Victoria!

Just a few nitpicks - to keep this styled like we would our source code, so that order looks like:

1. properties
2. inits
3. functions

Reply:
addressed ae732e8b2187cf2b561bad7b601c611e573bb23a